internal/nice/nice.go

```diff
@@ -0,0 +1,25 @@
+// package nice provides nice printing for the CLI
+package nice
+
+import (
+	"fmt"
+
+	ct "github.com/daviddengcn/go-colortext"
+)
+
+// PrintKyma prints the Kyma word with its identity color
+func PrintKyma() {
```

Comment:
Will this colorful output work in Windows?

Reply:
According to the docs at https://github.com/daviddengcn/go-colortext it already handles both unix and windows terminals accordingly.

Reply:
Tested on power shell and it works :)